### DISCUSSION TITLE
classic xbox

### DISCUSSION THREAD

**Discussion by Criosbolano** [darthcloud/BlueRetro#181] [Q&A] [answered]
Good day, I would like to know if this wireless meter is compatible with the Classic or original XBOX console?

**Comment by darthcloud** [comment, marked as answer]
No

**Reply by davidxgames** [reply]
no? https://www.youtube.com/watch?v=F9nkR7b0eCI

**Comment by pmgducati** [comment]
@Criosbolano if you haven't already, look into the ogx360.  It's the best wireless solution for the original Xbox that I have found.  I built one last year and love it.  You can also buy them pre made if you do some googling 🙂

https://github.com/Ryzee119/ogx360

**Comment by davidxgames** [comment]
> Good day, I would like to know if this wireless meter is compatible with the Classic or original XBOX console?

yes? https://www.youtube.com/watch?v=F9nkR7b0eCI

**Reply by pmgducati** [reply]
The guy in that video above just dropped a video on how he did that with the original Xbox. He's modifying a Xbox to PS2 adapter to enable the Bluetooth connectivity. You would need four, ESP 32s and four PS2 to Xbox adapters to be able to get four players. Which really puts you no different spot than the OG x360

**Reply by darthcloud** [reply]
I wouldn't say modify, He simply using BlueRetro in PS2 mode and connect it to the PS2-XBOX adapter. ;)

Daisy chaining adapters like that add additional latency.

**Comment by darthcloud** [comment]
@jimmyh98 BlueRetro is using SPI HW for PS2, so that wouldn't change anything.
